middleware/http/client_test.go

```diff
@@ -0,0 +1,52 @@
+package http_test
+
+import (
+	"net/http"
+	"testing"
+
+	zipkin "github.com/openzipkin/zipkin-go"
+	httpmiddleware "github.com/openzipkin/zipkin-go/middleware/http"
+	"github.com/openzipkin/zipkin-go/reporter/recorder"
+)
+
+func TestMiddleware(t *testing.T) {
```

Comment:
does.. TestHttpClientMiddleware clarify anything, or during runs it is inferred from the source file name..?

Reply:
Name will probably change to be less generic as i expect to break out parts and scenarios to be tested separately.